sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -180,8 +186,8 @@ function continue_enqueuing( $configs = null, $enqueue_status = null ) {
 		 * @since 4.2.0
 		 * @since 7.3.0 Added $range arg.
 		 *
-		 * @param args ''
-		 * @param $range array 
+		 * @param string $arg
+		 * @param array  $range
```

Comment:
Same as above.

Reply:
👍 Addressed in d8af90be351f13678f2b5368ca13ec37bef07d1c